Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Search/Filter.cpp

```diff
@@ -404,6 +404,29 @@ namespace AzToolsFramework
             return entry->GetEntryType() == m_entryType;
         }
 
+        //////////////////////////////////////////////////////////////////////////
+        // AssetPathFilter
+        //////////////////////////////////////////////////////////////////////////
+        AssetPathFilter::AssetPathFilter()
+        {
+        }
+
+        void AssetPathFilter::SetAssetPath(AZ::IO::Path path)
```

Comment:
nit: Might be able to pass by `const AZ::IO::Path&`.

Reply:
Thanks, I'll try that out!